mcs/class/corlib/System.Reflection/MonoMethod.cs

```diff
@@ -229,7 +229,7 @@ public override MethodInfo GetBaseDefinition ()
 			return get_base_method (this, true);
 		}
 
-		internal override MethodInfo GetBaseMethod ()
+		internal MethodInfo GetBaseMethod ()
```

Comment:
Maybe move the `TODO` comment from reference source to here?
https://github.com/mono/mono/blob/d3af6d44ff0809e68a9cb98e5c5452a36bc2065f/mcs/class/referencesource/mscorlib/system/reflection/methodinfo.cs#L134-L135